autorest/serializers/operation_group_serializer.py

```diff
@@ -8,13 +8,34 @@ def __init__(self, code_model, env, operation_group, async_mode):
         self.async_mode = async_mode
         self._operation_group_file = None
 
+    @staticmethod
+    def _change_imports_for_async(imports):
+        # change model import for async operation groups
```

Comment:
Create a new instance, call a merge to create a local copy, and then mutate it as you do. Otherwise you are actually mutating the original one

Reply:
and return the new instance